backend/entities/ProjectSnapshot.cs

```diff
@@ -23,6 +23,31 @@ public class ProjectSnapshot : BaseEntity
         /// </summary>
         public Project Project { get; set; }
 
+        /// <summary>
+        /// get/set - The date that this snapshot was taken.
+        /// </summary>
+        public DateTime SnapshotOn { get; set; }
+
+        /// <summary>
+        /// get/set - Optional, a project snapshot id to compare to this snapshot for variance reporting.
+        /// </summary>
+        public int? FromSnapshotId { get; set; }
+
+        /// <summary>
+        /// get/set - Optional, a project snapshot to compare to this snapshot for variance reporting.
+        /// </summary>
+        public ProjectSnapshot FromSnapshot { get; set; }
+
+        /// <summary>
+        /// get/set - Whether this report is final and should not be deleted.
+        /// </summary>
+        public bool IsFinal { get; set; }
+
+        /// <summary>
+        /// get/set - An optional, descriptive name given to the report.
+        /// </summary>
+        public string Name { get; set; }
+
         #region Financials
         /// <summary>
         /// get/set - The netbook value which is the sum of the properties.
```

Comment:
Also should add the `Appraised` property

Reply:
Should we? Currently there is no way to add appraised during the disposal process.

Reply:
So this value will always be 0.

Reply:
The `Project` is getting updated this sprint to include `Appraised`